### PR TITLE
Change to README

### PR DESCRIPTION
**Pull request by mattbrown11**
Added steps to the README to resolve implicit dependencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,7 +216,7 @@ After the pipeline completes, you should find a file, `numbers.txt`, in your wor
 
 ### Python Processes
```
Comment (mattbrown11) [inline README.md]: We should probably make mention of the Python bindings to Vital.  Specifically, a good entry point is the self-documenting Jupyter Notebook located in /vital/bindings/python/Kwiver_Python_Demo.ipynb. Perhaps someone with a better feel for the overall architecture can weigh in on how/where this should go.

**Comment by linus-sherrill** [comment]
It is great to have a list of how to install all requirements needed to build fletch, but the description only addresses how to do this for one Ubuntu version. There should be a similar list of requirements for other supported systems (e.g. CentOS, Windows, os-X )

**Comment by mleotta** [comment]
I haven't had a chance to read this in great detail, but my first impression is that we should promote using Fletch as the official way to get the dependencies for KWIVER on all platforms.  There is still value in having instructions for building against system packages on Ubuntu 14.04 or some other platform, but I wouldn't put that first.  It would be better on a secondary page somewhere.  The Ubuntu instructions provide enough packages for a minimal build, but for KWIVER to be useful you really need VXL, OpenCV, Ceres, etc.   We don't want people trying this minimal build on Ubuntu first and then getting frustrated that it can't actually do anything useful.  Some packages, like VXL, don't have a modern enough version packaged for Ubuntu.

**Comment by mattbrown11** [comment]
The updates still assume Fletch is the primary provider of packages.  However, I just added notes on how to install some additional dependencies that Fletch doesn't currently provide.

Perhaps it makes sense to move this into the doc/manuals sub-directory.  I didn't realize until later that there already is an install_guide.rst in that directory.  Perhaps we could point to the system-specific install guides from the main README.

**Comment by hughed2** [comment]
Can one of the admins verify this patch?

**Comment by hughed2** [comment]
Can one of the admins verify this patch?

**Comment by hughed2** [comment]
Can one of the admins verify this patch?

**Comment by mattbrown11** [comment]
What else needs to be done to finish this out?

**Comment by mattbrown11** [comment]
@aaron-bray This is the KWIVER documentation branch I was working on.

**Comment by mleotta** [comment]
This PR is now outdated and I'm going to close it.  The material added here is still useful but should be added to new restructured text file (not the main README) that focuses on building KWIVER against native Debian/Ubuntu packages.  The recommended approach is building against Fletch, but there are cases where building against system packages is needed.

**Pull request closed without merging**